### ISSUE TITLE
[A11Y] Feedback item should be atleast 48dp in size

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3927]
While working on #3901 I had to `DisableAccessibilityChecks` on one test case because the test case was failing as shown below:
<img width="500" alt="Screenshot 2021-10-14 at 3 44 29 PM" src="https://user-images.githubusercontent.com/9396084/137300382-d69508c5-17bf-4c49-bbe3-45b4c37e7249.png">

Feedback item should be atleast 48dp in size. This is needed because Feedback items can have clickable content inside them.

Add screenshots of before /after UI under all possible combinations and make sure they are exactly same.
- Mobile Portrait
- Mobile Landscape
- Tablet Portrait
- Tablet Landscape

**Comment by bkaur-bkj** [comment]
@rt4914 sir I viewed the feedback.xml file , there we have  frame layout and a textview inside, textview presently does not have any margin, just padding dimens and frame layout has margins, so do I need to alter margin of frame layout and inc padding of textview?

**Comment by bkaur-bkj** [comment]
@rt4914 sir as we discussed over last meet, you suggested me to use padding of Framelayout to be extended to padding of textview as textview currently had no margin so we were not able to use margin to increase current padding.
But I viewed the file- feedback.xml and found that even Framelayout has not padding it simply has margins and if I try to use its margin to be converted or changed to padding of textview it will alter the UI in anyway I try. Do let me know what can be done.

**Comment by BenHenning** [comment]
@bkaur-bkj it's expected that the feedback item might get slightly taller as a result of this change since it can sometimes be smaller than 48dp. What UI change did you notice?

**Comment by Rohit1173** [comment]
I want to work on this issue
